src/modules/missionHelper/i18n/it_IT.json

```diff
@@ -0,0 +1,93 @@
+{
+	"diyMission": "Questa missione, sembra essere una missione dell\'Alleanza!",
```

Comment:
@LSSM DEVS. Here a little issue. 
het \ has to be deleted. I think

Reply:
Yes, because the "" the ' don't have to be escaped